src/main/java/dev/imabad/mceventsuite/spigot/modules/eventblocker/EventBlockListener.java

```diff
@@ -0,0 +1,116 @@
+package dev.imabad.mceventsuite.spigot.modules.eventblocker;
+
+import com.google.gson.JsonObject;
+import dev.imabad.mceventsuite.spigot.config.SpigotConfig;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.Event;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.block.*;
+import org.bukkit.event.entity.*;
+import org.bukkit.event.hanging.HangingPlaceEvent;
+import org.bukkit.event.inventory.*;
+import org.bukkit.event.player.*;
+import org.bukkit.event.raid.RaidTriggerEvent;
+import org.bukkit.event.world.StructureGrowEvent;
+
+import java.io.IOException;
+
+public class EventBlockListener implements Listener {
+
+    SpigotConfig configFile;
+    JsonObject config;
+
+    public EventBlockListener() {
+        try {
+            configFile = new SpigotConfig("config");
+            config = configFile.getJson();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    @EventHandler
+    public void onEvent(Event e) {
+        //If the configuration has the event
+        if (config.has("eventblocker." + e.getClass().getSimpleName().toLowerCase())) {
```

Comment:
Can we move `e.getClass().getSimpleName().toLowerCase()` into a variable so we don't need to keep calling it?